salt/modules/bigip.py

```diff
@@ -0,0 +1,1723 @@
+# -*- coding: utf-8 -*-
+'''
+An execution module which can manipulate an f5 bigip via iControl REST
+    :maturity:      develop
+    :platform:      f5_bigip_11.6
+'''
+
+# Import python libs
+import requests
```

Comment:
Ah, didn't realize requests dependency for salt was removed.  Good to know.  It would probably make more sense to utilize salt.utils.http then.